app/src/main/java/app/insti/api/RetrofitInterface.java

```diff
@@ -107,4 +113,28 @@
 
     @GET("search")
     Call<ExploreResponse> search(@Header("Cookie") String sessionID, @Query("query") String query);
-}
+
+    @GET("complaints")
+    Call<List<Venter.Complaint>> getAllComplaints(@Header("Cookie") String sessionId);
+
+    @GET("complaints?filter=me")
+    Call<List<Venter.Complaint>> getUserComplaints(@Header("Cookie") String sessionId);
+
+    @GET("complaints/{complaintId}")
+    Call<Venter.Complaint> getComplaint(@Header("Cookie") String sessionId, @Path("complaintId") String complaintId);
+
+    @PUT("complaints/{complaintId}")
```

Comment:
Note: Base URL has changed to `/api/venter` as of https://github.com/wncc/IITBapp/commit/ff12ca797401856dc43c51322d744597a6da220a

Reply:
Refer to the [docs](https://wncc.github.io/IITBapp/) if there is any confusion.